.git-blame-ignore-revs

```diff
@@ -1,2 +1,4 @@
 # Auto-format the whole project using OCamlformat
 75efc08df487fa1a89fb86e446aadc8f6273b5bb
+# ocamlformat 0.17.0
+b0c03d1b76f7da7d88d40bbc5029accfa9228112
```

Comment:
This feature of Git doesn't work well in my experience but it doesn't cost much to record these anyway.